scalardl-node-client-sdk.js

```diff
@@ -109,6 +109,31 @@ function _createMetadata(properties) {
   return metadata;
 }
 
+/**
+ * Read the content of file-based properties
+ * @param {Object} properties
+ * @return {Object}
+ */
+function _resolveFileBasedProperties(properties) {
+  const fs = require('fs');
+
+  properties['scalar.dl.client.cert_pem'] =
+    properties['scalar.dl.client.cert_pem'] ||
+    fs.readFileSync(properties['scalar.dl.client.cert_path']).toString();
+
+  properties['scalar.dl.client.private_key_pem'] =
+    properties['scalar.dl.client.private_key_pem'] ||
+    fs.readFileSync(properties['scalar.dl.client.private_key_path']).toString();
+
+  properties['scalar.dl.client.tls.ca_root_cert_pem'] =
+    properties['scalar.dl.client.tls.ca_root_cert_pem'] ||
+    fs.readFileSync(
+        properties['scalar.dl.client.tls.ca_root_cert_path'],
```

Comment:
Do we need `,` at the end?
Also, should this be formatted?

Reply:
Ah, that is because we use Google's ESLint config and it has this rule (comma-dangle)
https://github.com/google/eslint-config-google/blob/master/index.js#L195
It not only applies to the array and the object but also the function arguments.

Reply:
If it is formatted properly, It's fine!